docs/api/browser-window.md

```diff
@@ -139,7 +139,7 @@ state is `hidden` in order to minimize power consumption.
 * On Linux the type of modal windows will be changed to `dialog`.
 * On Linux many desktop environments do not support hiding a modal window.
 
-## Class: BrowserWindow
+## Class: BrowserWindow extends `BaseWindow`
```

Comment:
If we want to reduce the duplication while maintaining visibility, maybe we can make the docs generator output inherited methods in a new section on the page?

Reply:
@samuelmaddock I'd love to, but I'm not sure that should block this PR :)